### ISSUE TITLE
about the ”Flex-Rate-Hier-Bidir-Video-Compression“train script

### ISSUE THREAD

**Issue by luxiaotong321** [KUIS-AI-Tekalp-Research-Group/video-compression#9]
Thank you for your outstanding contribution! We intend to reproduce your model”Flex-Rate-Hier-Bidir-Video-Compression“,but we can not figure out some details in the training.Could you share the training script?

**Comment by WenBingo** [comment]
I'd like to ask if your question about LHBDC has been solved?

**Comment by luxiaotong321** [comment]
Hello, I can run the training code of LHBDC without any problem, but I always have problems when I write the train code of the paper “Flex-Rate-Hier-Bidir-Video-Compression”!

**Comment by WenBingo** [comment]
ok, thank you for your answer, maybe it's my  mistake

**Comment by makinyilmaz** [comment]
Dear luxiaotong321 I am sorry for the late reply. Can you please share the problems you have encountered regarding Flex-Rate-Hier-Bidir-Video-Compression so I could help ?

**Comment by luxiaotong321** [comment]
> Dear luxiaotong321 I am sorry for the late reply. Can you please share the problems you have encountered regarding Flex-Rate-Hier-Bidir-Video-Compression so I could help ?
I'd like to know how λ for loss works in the code in this paper, and when my calls to b_model are always terminated with a “cdf update” error, so could you please provide me with the "Flex-Rate-Hier-Bidir- Video-Compression" of the trian code I learn a little?

**Comment by makinyilmaz** [comment]
> > Dear luxiaotong321 I am sorry for the late reply. Can you please share the problems you have encountered regarding Flex-Rate-Hier-Bidir-Video-Compression so I could help ?
> > I'd like to know how λ for loss works in the code in this paper, and when my calls to b_model are always terminated with a “cdf update” error, so could you please provide me with the "Flex-Rate-Hier-Bidir- Video-Compression" of the trian code I learn a little?

λ controls the rd tradeoff. We used the approach proposed in the following work: "Asymmetric gained deep image compression with continuous rate adaptation". I will be releasing a training script for our ICIP2024 paper in a week. You may check it and use it on other models as well.

**Comment by luxiaotong321** [comment]
> > > Dear luxiaotong321 I am sorry for the late reply. Can you please share the problems you have encountered regarding Flex-Rate-Hier-Bidir-Video-Compression so I could help ?
> > > I'd like to know how λ for loss works in the code in this paper, and when my calls to b_model are always terminated with a “cdf update” error, so could you please provide me with the "Flex-Rate-Hier-Bidir- Video-Compression" of the trian code I learn a little?
> 
> λ controls the rd tradeoff. We used the approach proposed in the following work: "Asymmetric gained deep image compression with continuous rate adaptation". I will be releasing a training script for our ICIP2024 paper in a week. You may check it and use it on other models as well.

Thank you so much, your help is much appreciated!!!!

**Comment by makinyilmaz** [comment]
Hello again. I am sorry for the late update. Now I have shared a training script under ICIP2024/src. It is not perfect but it may help you to train your own models. I am closing the issue  for now. If you need, you may always create new issues and I will try my best to help you guys. Best
